source/index.html.md

```diff
@@ -314,6 +344,21 @@ curl "https://write.as/api/posts/rf3t35fkax0aw" \
   -d '{"token": "ozPEuJWYK8L1QsysBUcTUKy9za7yqQ4M", "body": "My post is updated."}'
 ```
 
+```
```

Comment:
Looks like we need to add `swift` here, like on the other examples, to get the correct syntax highlighting.

Reply:
Fixed in 8a0b9e5.